server/middleware/populateUserAuthorities.ts

```diff
@@ -1,13 +1,18 @@
 import { RequestHandler } from 'express'
 import { jwtDecode } from 'jwt-decode'
-import { ApplicationRoles } from './roleBasedAccessControl'
+import ApplicationRole from '../enums/applicationRole'
+import ApplicationAction from '../enums/applicationAction'
+import { userHasPermissionTo } from './roleBasedAccessControl'
 
 export default function populateUserAuthorities(): RequestHandler {
   return async (req, res, next) => {
     if (res.locals?.user?.token) {
       const { authorities: roles = [] } = jwtDecode(res.locals.user.token) as { authorities?: string[] }
 
-      res.locals.hasEditAuthority = roles.includes(ApplicationRoles.ROLE_EDUCATION_WORK_PLAN_EDITOR)
+      // hasEditAuthority is deprecated - use userHasPermissionTo instead
+      res.locals.hasEditAuthority = roles.includes(ApplicationRole.ROLE_EDUCATION_WORK_PLAN_EDITOR)
```

Comment:
`hasEditAuthority` will be nuked once we've migrated everything over to the new `userHasPermissionTo` function
But at the moment it is used in a lot of places, so we need to keep it for now